<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>688</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEditArr">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>481</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditQ">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditK">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>50</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonB5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>501</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Задана целочисленная последовательность A=(ai), i=1..n, n&lt;=100.
Назовем серией группу подряд идущих одинаковых элементов,
а длиной серии — количество этих элементов (длина серии может быть равна 1).
Вывести массив, содержащий длины всех серий исходного массива. </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSort">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>431</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Задана целочисленная последовательность A=(ai), i=1..n,
 n&lt;=100. Если она является перестановкой, 
то есть содержит все числа от 1 до N, 
то вывести сообщение, в противном случае
 вывести номер первого недопустимого элемента.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonB10">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>190</y>
      <width>371</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Заданы коэффициенты многочлена 
Рn = a0 + a1 * x1 + a2 * x2 + …an * xn.
Найти К - тую производную многочлена.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonApp">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>330</y>
      <width>431</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Даны  два  неубывающих массива X=(xi),i=1..n, n&lt;=10, 
 и Y=(yi),i=1..m, m&lt;=10 и  число q. Найти сумму вида (x(i)+y(j)),
 наиболее близкую к числу q. (Число действий порядка m+n,
 дополнительная память - фиксированное число целых переменных, 
сами массивы менять не разрешается.)</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditRes">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>500</y>
      <width>541</width>
      <height>85</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>491</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Введите массив:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Введите число(q):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Введите порядок производной:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>450</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Результат:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>580</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Подготовил Лу Тимур 5 группа</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1030</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
